test/1_stdlib/StaticString.swift

```diff
@@ -15,7 +15,7 @@ StaticStringTestSuite.test("PointerRepresentation/ASCII/Empty") {
     utf8 in
     expectEqual(0, utf8.count)
   }
-  expectEqual("", String(str))
+  expectEqual("", str.description)
```

Comment:
Nobody should ever call `.description` (except for one place in reflection machinery).

I think StaticString should be lossless string convertible.